environment-api/src/main/java/com/sequenceiq/environment/api/v1/credential/model/request/EditCredentialRequest.java

```diff
@@ -0,0 +1,56 @@
+package com.sequenceiq.environment.api.v1.credential.model.request;
+
+import javax.validation.Valid;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.sequenceiq.authorization.annotation.ResourceObjectField;
+import com.sequenceiq.authorization.resource.AuthorizationResourceAction;
+import com.sequenceiq.authorization.resource.AuthorizationVariableType;
+import com.sequenceiq.environment.api.doc.ModelDescriptions;
+import com.sequenceiq.environment.api.doc.credential.CredentialDescriptor;
+import com.sequenceiq.environment.api.doc.credential.CredentialModelDescription;
+import com.sequenceiq.environment.api.v1.credential.model.CredentialBase;
+import com.sequenceiq.environment.api.v1.credential.model.parameters.azure.AzureCredentialRequestParameters;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel(description = CredentialDescriptor.CREDENTIAL_NOTES, parent = CredentialBase.class, value = "CredentialV1Request")
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(Include.NON_NULL)
+public class EditCredentialRequest extends CredentialBase {
+
+    @ApiModelProperty(value = ModelDescriptions.NAME, required = true)
```

Comment:
would we like to allow update the name of a credential?
I think we should add constraint at least for the size as it is VARCHAR(255) in the database and creation requests allow only 100 chars as maximum length